### PR TITLE
Add page for contributing

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
cc @networkException @Redecorating

File: docs/donations.md
Comment (networkException) [inline docs/donations.md:3]: I don't think a `,` is grammatically correct here. Also perhaps this could be phrased a little differently, how about "The T2 project itself currently does not accept finational donations. However, some of the contributors and maintainers of the project to accept donations on their own."?

File: docs/index.md
Comment (networkException) [inline docs/index.md:24]: Can you reword this to "Visit the donations page for more details."? It's better to be transparent about where a link leads to before clicking

File: mkdocs.yml
Comment (networkException) [inline mkdocs.yml:42]: This is a bit akward, we should decide on either Contribute or donations

**Review by networkException — CHANGES_REQUESTED** [review]

File: mkdocs.yml
Comment (AdityaGarg8) [reply mkdocs.yml]: db0f6f2

File: docs/index.md
Comment (AdityaGarg8) [reply docs/index.md]: db0f6f2

File: docs/donations.md
Comment (AdityaGarg8) [reply docs/donations.md]: db0f6f2

**Comment by AdityaGarg8** [comment]
If anyone else is accepting donations, please mention it here.

**Review by networkException — APPROVED** [review]

**Pull request merged**
